cypress/support/commands.js

```diff
@@ -24,3 +26,43 @@ Cypress.Commands.add('awaitLoad', (divIds) => {
         .and('eq', '0');
   });
 });
+
+/**
+ * Cypress tests have two phases. In the first phase, each line is executed, top
+ * to bottom, as usual. However, any `cy.*` commands, like `cy.get()`, do not
+ * actually do anything when they execute. Instead, they enqueue a command into
+ * an internal Cypress "queue". After all lines have been executed, Cypress
+ * knows the full queue. At that point, the "Cypress phase" starts. Cypress
+ * actually executes each command in the queue. After each command completes, it
+ * will execute any `.then` blocks that were chained onto it. Thus, in the
+ * following program:
+ * ```
+ * console.log('message 1');
+ * cy.wrap(console.log('message 2')).then(() => console.log('message 4'));
+ * console.log('message 3');
+ * cy.get('#id').then(() => console.log('message 5'));
+ * ```
+ * The messages appear in the specified order.
+ *
+ * In the vast majority of cases, all important test instructions should be
+ * executed during the Cypress phase, *not* during the initial test execution
+ * phase. For the most part, that can be accomplished by chaining statements off
+ * a previous Cypress command. However, for readability, it is nice to be able
+ * to call functions directly, as opposed to having long chain blocks. For that
+ * reason, we have a utility function, `queue`, that will put the given `lambda`
+ * as a `.then` block after a `cy.wrap(null)` command. That will ensure that the
+ * `lambda` only executes during the Cypress phase.
+ *
+ * As a convention in this codebase, a function that returns a {@link
+ * Cypress.Chainable} can be assumed to execute its instructions during the
+ * Cypress phase. Only functions that do not return a {@link Cypress.Chainable}
+ * execute during the initial phase.
+ *
+ * For more information, see
+ * https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Commands-Are-Asynchronous
+ * @param {Function} lambda Code to be executed during Cypress phase.
+ * @return {Cypress.Chainable<any>}
+ */
+function cyQueue(lambda) {
```

Comment:
So would it be accurate to say it only makes sense to call this function as the first thing in a method? (test or utility?)

Reply:
Good point, added.